bcda/cclf/parser_test.go

```diff
@@ -293,3 +293,30 @@ func TestGetCCLFMetadata(t *testing.T) {
 		})
 	}
 }
+
+func TestCheckIfAttributionCSVFile(t *testing.T) {
+	tests := []struct {
+		name      string
+		path      string
+		testIsCSV bool
+	}{
+		{name: "Is an Attribution CSV File path", path: "P.PCPB.M2014.D00302.T2420001", testIsCSV: true},
+		{name: "Is not an Attribution CSV File path (incorrect first)", path: "T.PCPB.M2014.D00302.T2420001", testIsCSV: false},
```

Comment:
By incorrect first do you mean the first part of the file name (in this case the "T")?  A T is actually an acceptable character here as far as I understand (it can be a "T" or a "P" for test and prod).

Reply:
Updated.